cookbooks/aws-parallelcluster-slurm/recipes/test_resource.rb

```diff
@@ -0,0 +1,10 @@
+# This is a helper recipe executing the default action of the resource
+# which type was set as `resource` attribute during the test execution.
+# This is equivalent to:
+# resource_type 'test'
+# or
+# resource_type 'test' do
+#   action :default_action
+# end
+
+declare_resource(node['resource'].to_sym, 'test')
```

Comment:
Please double check if this is required.

I added this for simmetry ..but I am successfully using "aws-parallelcluster::test_resource" for testing...